src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +17,13 @@
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .flatMap(s -> Arrays.stream(s.split(",")))
+                .map(s1 -> Integer.valueOf(s1))
```

Comment:
```suggestion
                .map(s -> Integer.valueOf(s))
```
it's okay to use same shortcut names for variables in streams and it's considered bad practice to use numbers in variable names